Lib/getpass.py

```diff
@@ -40,6 +43,9 @@ def unix_getpass(prompt='Password: ', stream=None):
 
     Always restores terminal settings before returning.
     """
+    if not _is_ascii(echochar):
+        return ValueError(f"'echochar' must be ASCII, got: {echochar!r}")
```

Comment:
An alternative is to have `_check_echochar(echochar)` which would raise the ValueError.